src/SSHDebugPS/UI/ContainerPickerDialogWindow.xaml

```diff
@@ -292,154 +405,146 @@
-                Grid.Column="1"
+                Margin="12,4,0,0"
+                MinWidth="75"
+                Height="23"
                 Command="{Binding AddSSHConnectionCommand}"
                 CommandParameter="{Binding}"
-                IsEnabled="{Binding AddSSHConnectionCommand.CanExecute}">
-            <vsimaging:CrispImage Moniker="{x:Static catalog:KnownMonikers.Add}"
-                                  Width="16"
-                                  Height="16">
-                <Image.ToolTip>
-                    <TextBlock Text="{Binding Path=AddSSHConnectionCommand.ToolTip, Mode=OneWay}" />
-                </Image.ToolTip>
-            </vsimaging:CrispImage>
-        </Button>
+                IsEnabled="{Binding AddSSHConnectionCommand.CanExecute}"
+                Content="{x:Static local:UIResources.AddNewSSHConnectionButtonLabel}"
+                ToolTip="{x:Static local:UIResources.AddNewSSHConnectionToolTip}"/>
 
-        <!-- Advanced Settings -->
-        <Expander x:Name="advancedSettingsExpander"
-                  x:Uid="uidadvancedSettingsExpander"
-                  HorizontalAlignment="Stretch"
-                  Header="{x:Static local:UIResources.AdvancedSettingsHeader}"
-                  Grid.Row="2"
-                  Grid.ColumnSpan="2"
-                  Margin="0,10,0,0"
-                  VerticalAlignment="Top"
-                  ExpandDirection="Down">
-            <Grid>
-                <Grid.RowDefinitions>
-                    <RowDefinition Height="Auto" />
-                </Grid.RowDefinitions>
-                <Grid.ColumnDefinitions>
-                    <ColumnDefinition Width="Auto" />
-                    <ColumnDefinition Width="*" />
-                </Grid.ColumnDefinitions>
-                
-                <Label x:Uid="uidHostnameLabel"
-                       Grid.Row="0"
-                       Grid.Column="0"
-                       x:Name="Hostname"
-                       Target="{Binding ElementName=hostnameTextBox}"
-                       Margin="19,5,0,5"
-                       Content="{x:Static local:UIResources.HostnameLabel}"
-                       ToolTip="{x:Static local:UIResources.HostnameTip}" />
+        <!-- Hostname label and textbox -->
 
-                <TextBox x:Uid="uidHostnameTextBox"
-                         Grid.Row="0"
-                         Grid.Column="1"
-                         x:Name="hostnameTextBox"
-                         Text="{Binding Path=Hostname, Mode=TwoWay,UpdateSourceTrigger=PropertyChanged}"
-                         PreviewKeyDown="HostnameTextbox_PreviewKeyDown"
-                         Margin="0,5,0,5"
-                         Padding="2" />
-                </Grid>
-        </Expander>
+        <Label Grid.Row="2"
+               Grid.ColumnSpan="2"
+               x:Uid="uidHostnameLabel"
+               x:Name="HostnameLabel"
+               Margin="0,9,0,0"
+               Padding="0"
```

Comment:
I think you can remove this too

Reply:
This isn't the default which is why i set it to 0